docs/docs/guides/customize-identity-model.mdx

```diff
@@ -0,0 +1,304 @@
+  make sure you are done before saving the Identity Model.
+
+You can specify a title for your custom Identity Model. The default is
+`"title": "Person",` but this can be changed to something that describes this
+Identity Model more accurately, e.g.
+
+```
+"title": "Customer",
+"title": "Administrator",
+"title": "Foo",
+etc...
+```
+
+## Customize Fields
+
+An Identity Models traits are specified under
+
+```json
+"traits": {
+"type": "object",
+"properties": {
+```
+
+Each trait translates into a field on the user-facing frontend. For example the
+"username & password" preset defines two traits - username and password:
+
+```json
+// username & password identity model
+    "traits": {
+      "type": "object",
+      "properties": {
+        "username": {
+          "type": "string",
+          "title": "Username",
+          "ory.sh/kratos": {
+            "credentials": {
+              "password": {
+                "identifier": true
+              }
+            },
+            "recovery": {
+              "via": "email"
+            },
+            "verification": {
+              "via": "email"
+            }
+          }
+        }
+      },
+      "required": [
+        "username"
+      ],
+      "additionalProperties": false
+    }
+```
+
+This Identity Model translates into the following sign-up screen:
+![Username Password Identity Model](../images/identity-model/username-password.png)
+
+The part highlighted below defines the identities `username` for the
+username+password flow in the Ory Cloud. It also includes a method for
+[`recovery`](https://www.ory.sh/kratos/docs/self-service/flows/account-recovery#recovery-link-method)
+as well as
+[`verification`](https://www.ory.sh/kratos/docs/self-service/flows/verify-email-account-activation/#verification-link-method).
+Currently, only the `email` method is available (recovery/verification via a link
+sent in an email).
+
+```diff
+    "traits": {
+      "type": "object",
+      "properties": {
++       "username": {
++         "type": "string",
++         "title": "Username",
++         "ory.sh/kratos": {
++           "credentials": {
++             "password": {
++               "identifier": true
++             }
++           },
+            "recovery": {
+              "via": "email"
+            },
+            "verification": {
+              "via": "email"
+            }
+          }
+        }
+      },
+      "required": [
+        "username"
+      ],
+      "additionalProperties": false
+    }
+```
+
+To add traits to the Identity Model, add them inside the `traits "properties"`
+
+```json
+"traits": {
+"type": "object",
+"properties": {
+// your identities traits go here
+       }
+    }
+```
+
+for example the GitHub Handle as `string` :
+
+```json
+"traits": {
+"type": "object",
+"properties": {
+        "handle": {
+          "type": "string",
+          "title": "Your GitHub Handle"
+        }
+      }
+    }
+```
+
+or a mobile phone number as `integer`:
```

Comment:
phone number as an integer won't be a good user experience.
It will create a input field with a "increase/decrease" button on the side - which looks bad.

Reply:
True, I noticed that as well. 
I think `age` would probably a good example for now.